scripts/coverage.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -e
+
+rm -rf coverage
+mkdir -p coverage/workspaces
+for l in $(ls packages); do
+    if [ -f packages/$l/coverage/coverage-final.json ]; then
+        cp packages/$l/coverage/coverage-final.json coverage/workspaces/$l-coverage-final.json
+    fi
+done
+yarn run nyc merge coverage/workspaces coverage/monorepo-coverage.json
+yarn run nyc report -t coverage --report-dir coverage/html --reporter=html-spa
```

Comment:
nit: we have a backlog to convert our .sh scripts to typescript
```
"get-coverage": yarn ts-node ./myscript.ts
```
You can import dependencies via scripts/package.json, such as
glob/fs and path normalizers (join) for cross-platform compatibility
and we even have a tool to run 'bash' commands for everything that doesn't translate over via execa:
```
execa.sync('git', ['reset', '--hard', 'HEAD'], {
    cwd: path.resolve(__dirname, '../'),
  });
  ```